Cargo.toml

```diff
@@ -2,10 +2,11 @@
 
 [package]
 name = "prodzilla"
-version = "0.1.0"
+version = "0.0.3-rc.5"
 edition = "2021"
 
 [dependencies]
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
I didn't have openssl installed and so get an error on build:

```
Error configuring OpenSSL build:
...
Failed to execute: program not found
```

I think it's because I'm on Windows and don't have openssl installed. I would prefer if the app didn't require openssl if not needed, but if it does I think it might be worth making a note in the readme?

Reply:
Having openssl in my path I now get
```
error: failed to run custom build command for `openssl-sys v0.9.102`

Caused by:
  process didn't exit successfully: `C:\Users\jorda\dev\prodzilla\target\debug\build\openssl-sys-340412e7b060235b\build-script-main` (exit code: 101)
  --- stdout
  cargo:rerun-if-env-changed=X86_64_PC_WINDOWS_MSVC_OPENSSL_NO_VENDOR
  X86_64_PC_WINDOWS_MSVC_OPENSSL_NO_VENDOR unset
  cargo:rerun-if-env-changed=OPENSSL_NO_VENDOR
  OPENSSL_NO_VENDOR unset
  running "perl" "./Configure" "--prefix=C:/Users/jorda/dev/prodzilla/target/debug/build/openssl-sys-f1431536b46b0606/out/openssl-build/install" "--openssldir=SYS$MANAGER:[OPENSSL]" "no-dso" "no-shared" "no-ssl3" "no-tests" "no-comp" "no-zlib" "no-zlib-dynamic" "--libdir=lib" "no-md2" "no-rc5" "no-weak-ssl-ciphers" "no-camellia" "no-idea" "no-seed" "no-capieng" "no-asm" "VC-WIN64A"

  --- stderr
  Can't locate Locale/Maketext/Simple.pm in @INC (you may need to install the Locale::Maketext::Simple module) 
```

It's my fault for using Windows lol, will try to debug

Reply:
I think the OpenSSL dependency is coming from `reqwest`, it only showed up as an issue where I needed to explicitly vendor it when I was trying to figure out cross-compilation. We should definitely get it compiling on Windows though, I'll see if I can figure it out 🤔

Reply:
Yeah weirdly it compiled + ran fine when I removed the `vendored` part from the Cargo.toml. Does it need the vendor feature?

Keen to merge it all once this is resolved  :)